cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -50,6 +50,7 @@
     if node.chef_environment == 'staging' && node.name == 'staging' # 'real' staging only
       cronjob at:'@reboot', do:home_dir('.dropbox-dist', 'dropboxd')
       cronjob at:'*/5 * * * *', do:deploy_dir('bin', 'cron', 'import_google_sheets')
+      cronjob at:'* * * * *', do:deploy_dir('bin', 'cron', 'sync_dropbox')
```

Comment:
minimum cron granularity is 1 min. The suggested way to get around this is to add duplicate jobs with sleeps to space them out.